src/lib/components/composites/paper-components/paper-view/PaperNavigationButton.svelte

```diff
@@ -2,47 +2,157 @@
     import ArrowLeft from "lucide-svelte/icons/arrow-left";
     import ArrowRight from "lucide-svelte/icons/arrow-right";
     import Tooltip from "../../utils/Tooltip.svelte";
-    import { goto } from "$app/navigation";
+    import type { Project, Project_Paper } from "$lib/model/api/project";
+    import { reviewMode } from "$lib/global-state/review-mode-state.svelte";
+    import { backendService } from "$lib/grpc-api";
+    import { toast } from "svelte-sonner";
+    import { navigatePaper } from "$lib/utils/paper-navigation";
+    import LoaderCircle from "lucide-svelte/icons/loader-circle";
 
     interface Props {
         direction: "left" | "right";
-        href: string;
-        onClick?: () => void;
+        loadingProject?: Promise<Project>;
+        loadingProjectPaper: Promise<Project_Paper | undefined>;
+        loading?: boolean;
+        paperQueue?: Project_Paper[];
+        nextProjectPaper?: Project_Paper;
+        previousProjectPaper?: Project_Paper;
     }
 
-    const { direction, href, onClick }: Props = $props();
+    let {
+        direction,
+        loadingProject,
+        loadingProjectPaper,
+        loading = $bindable(false),
+        paperQueue = $bindable([]),
+        nextProjectPaper = $bindable(undefined),
+        previousProjectPaper = $bindable(undefined),
+    }: Props = $props();
+    let buttonLeftDisabled: boolean = $state(true);
+    let buttonRightDisabled: boolean = $state(true);
+    let showLoadingSpinner = $state(false);
     const tooltipText = direction === "left" ? "Previous Paper" : "Next Paper";
+
+    async function getNextProjectPaperToReview(paper: Project_Paper) {
+        await backendService
+            .getNextPaperToReview({ id: paper.id })
+            .response.then((nextPaper) => {
+                nextProjectPaper = nextPaper;
+            })
+            .catch((error) => {
+                nextProjectPaper = undefined;
+                if (error.message !== "No%20next%20paper%20available.") {
+                    toast("Error, while loading the next paper:" + error);
```

Comment:
```suggestion
                    console.error("Error while loading the next paper:" + error);
                    toast("Error while loading the next paper);
```

Same for the ones below.
I thought we said we still include the console.error part.
And imo, the error itself should not be printed into the toast, as this is only for the user.

Reply:
You are right. This was to get the error messages easier during the e2e tests, but is now unnecessary.

Reply:
You now only need to remove the `:` at the end of the toast message.

Reply:
In my code, the `:` is still there. Also in the next functions.

Reply:
I don't know what happened. I was sure I have added them. But now it should be.